### PR TITLE
add static obstacles

### PR DESCRIPTION
**Pull request by MingfeiCheng**
Apollo does not show static obstacles, such as traffic cones. Add static obstacles.

### CODE REVIEW

**Comment by yuxip** [comment]
Thanks a lot for the PR, @MingfeiCheng ! Reviewing and testing...

File: src/carla_cyber_bridge/static.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from carla_cyber_bridge.traffic_participant import TrafficParticipant
+
+from cyber.carla_bridge.carla_proto.proto.carla_marker_pb2 import ColorRGBA
+from modules.perception.proto.perception_obstacle_pb2 import PerceptionObstacle
+
+
+class Static(TrafficParticipant):
+
+    """
+    Actor implementation details for vehicles
```
Comment (yuxip) [inline src/carla_cyber_bridge/static.py:22]: "for static objects" instead of "for vehicles"

File: src/carla_cyber_bridge/static.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        :param uid: unique identifier for this object
+        :type uid: int
+        :param name: name identiying this object
+        :type name: string
+        :param parent: the parent of this
+        :type parent: carla_cyber_bridge.Parent
+        :param node: node-handle
+        :type node: carla_cyber_bridge.CarlaCyberBridge
+        :param carla_actor: carla vehicle actor object
```
Comment (yuxip) [inline src/carla_cyber_bridge/static.py:37]: forgot to update the comment when copied over from vehicle.py?

File: src/carla_cyber_bridge/static.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        :type carla_actor: carla.Vehicle
```
Comment (yuxip) [inline src/carla_cyber_bridge/static.py:38]: same here

File: src/carla_cyber_bridge/static.py (hunk 2 above)
Comment (MingfeiCheng) [reply src/carla_cyber_bridge/static.py]: Sorry forgot to update the comment... I have updated the static.py. :)

File: src/carla_cyber_bridge/object_sensor.py [hunk 4]
```diff
@@ -89,4 +90,6 @@ def update(self, frame, timestamp):
                     cyber_objects.perception_obstacle.append(actor.get_object_info())
                 elif isinstance(actor, Walker):
                     cyber_objects.perception_obstacle.append(actor.get_object_info())
+                elif isinstance(actor, Static):
```
Comment (yuxip) [inline src/carla_cyber_bridge/object_sensor.py:93]: The comment at line 85 needs to be updated as well.

**Review by yuxip — COMMENTED** [review]
I've tested the code and everything seems to be ok. 

Just need a clarification, so from now on all of carla's [static objects](https://carla.readthedocs.io/en/latest/bp_library/#static) will be recognized by the pseudo objects sensors and related info are written to some "/apollo/perception/object_sensor" topic, correct? If so, we assume any consumer of this topic (e.g., PnC module to be tested) knows how to handle these static obstacles, is this the overall logic?

**Comment by yuxip** [comment]
with this PR static objects will show up in Dreamview as magenta boxes with yellow remarks, as the following
stationary_objects (these are some chairs and tables). And related info will be written by the object sensor to the  "/apollo/perception/object_sensor" topic.

File: src/carla_cyber_bridge/object_sensor.py (hunk 4 above)
Comment (yuxip) [reply src/carla_cyber_bridge/object_sensor.py:93]: once this comment is updated we are ready to merge:)

**Comment by MingfeiCheng** [comment]
> I've tested the code and everything seems to be ok.
> 
> Just need a clarification, so from now on all of carla's [static objects](https://carla.readthedocs.io/en/latest/bp_library/#static) will be recognized by the pseudo objects sensors and related info are written to some "/apollo/perception/object_sensor" topic, correct? If so, we assume any consumer of this topic (e.g., PnC module to be tested) knows how to handle these static obstacles, is this the overall logic?

Yes. The static actors are identified by 'static.prop' in carla_actor.type_id. As shown in [static objects](https://carla.readthedocs.io/en/latest/bp_library/#static), 'static.prop' can be used to identify all static actors in carla now. 

The info of Vehicles, Walkers and Static Obstacles will be written to '/apollo/perception/obstacles' based on the PerceptionObstacle proto in Apollo (modules.perception.proto.perception_obstacle_pb2). So, I just make sure that the Apollo can consume this information to get ground truth perception.

**Comment by yuxip** [comment]
Thanks for the explanation and the PR, @MingfeiCheng !

**Review by yuxip — APPROVED** [review]

**Pull request merged**